src/sfpowerkitCommand.ts

```diff
@@ -0,0 +1,71 @@
+import { SfdxCommand } from "@salesforce/command";
+import SFPKLogger, {COLOR_HEADER, LoggerLevel} from "./SFPKLogger";
+
+/**
+ * A base class that provides common funtionality for sfpowerscripts commands
+ *
+ * @extends SfdxCommand
+ */
+export default abstract class SFPowerkitCommand extends SfdxCommand {
+  public static isJsonFormatEnabled: boolean;
+  public static logLevel;
+  private sfpowerkitConfig;
+
+  /**
+   * Command run code goes here
+   */
+  abstract execute(): Promise<any>;
+
+  /**
+   * Entry point for the commands
+   */
+  async run(): Promise<any> {
+    // Always enable color by default
+    if (process.env.SFPOWERKIT_NOCOLOR) SFPKLogger.disableColor();
+    else SFPKLogger.enableColor();
+
+    this.setLogLevel();
+
+    for (const plugin of this.config.plugins) {
+      if (plugin.name === "sfpowerkit") {
+        this.sfpowerkitConfig = plugin;
+        }
+      }
+
+
+    SFPKLogger.log(
```

Comment:
There needs to be an environment variable that is used to hide the header.

Reply:
added the environment variable conditional